packages/wallet/frontend/src/components/Menu.tsx

```diff
@@ -232,6 +251,16 @@ export const Menu = () => {
               )
             )}
           </div>
+          <div className="hidden md:block">
+            <Link
```

Comment:
Considering that this is not a navigation, the component should not be a `Link`.